### PR TITLE
Remove duplicate cql execution

### PR DESCRIPTION
**Pull request by mickohanlon23**
@birick1 created this branch to prevent the Dashboard from firing CDS twice. This addresses the issue described in [CCSMCDS-215](https://jira.mitre.org/browse/CCSMCDS-215).

### CODE REVIEW

File: src/features/DecisionAids/Recommendations/index.js [hunk 1]
```diff
@@ -89,6 +90,8 @@ function Recommendations(props) {
         </Card.Body>
       </Card>
       <RiskEstimates input={riskTable} />
+      <br/>
+      <p>CDS Apply Time: {applyTime}ms </p>
```
Comment (mrnosal) [inline src/features/DecisionAids/Recommendations/index.js:95]: This was part of an earlier branch that added the CDS Apply time to the bottom of the Recommendation as an aid to performance debugging. Do we still want to include this?

**Review by mrnosal — COMMENTED** [review]
Do we want to expose the CDS Apply Time in the UI?

**Comment by mrnosal** [comment]
See new PR #67 which is a cherry-pick of Brian's commit. If that's approved, this PR (and branch) can be closed.

**Comment by mickohanlon23** [comment]
This can be closed now that https://github.com/ccsm-cds-tools/ccsm-cds-dashboard/pull/67 was approved and merged.

**Pull request closed without merging**